dashboard/client/src/common/ProfilingLink.tsx

```diff
@@ -96,3 +142,127 @@ export const CpuProfilingLink = ({
     </Link>
   );
 };
+
+export const ProfilerButton = ({
```

Comment:
can you add a component test for this component?

Reply:
added